### PR TITLE
fix: refactor for using tilt to start basic stack

### PR DESCRIPTION
**Pull request by ESmithaaS**
This pull request does the following:
- localizes the manifests for the otel-operator, cert-manager, jaeger-operator, and all the applications located in the "apps/default" folder.
- creates new and separate Tiltfiles for the otel-operator, cert-manager, jaeger-operator, and all the applications located in the "apps/default" folder.
- Remove the Tiltfile from the root of the repository.
- Creates the ability to spin up either a basic stack for observability or a ECK stack.  
-  update the README.md file to reflect the changes in the makefile and the new commands to spin up the otel-basic and otel-eck clusters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ NOTE: Requires k3d<br><br>
 
 Tilt takes care of creating resouces and giving you access to the logs, as well as creating any port-forwarding you need. You'll have easy access from tilt's builtin dashboard.<br>
 
-To spin up a k3d observability cluster, and deploy the default LGTM stack with tilt; run `make tilt-up`. <br>
+To spin up a k3d otel-basic cluster, and deploy the default LGTM stack with tilt; run `make tilt-basic`. <br>
```
Comment (adrielp) [inline README.md:45]: This is more of a note for myself, we'll probably want to change the `basic` term to something else. Doesn't need to be done here though. Will be a future iteration.

File: apps/default/Tiltfile [hunk 2]
```diff
@@ -0,0 +1,61 @@
+if k8s_context() != "k3d-otel-basic":
+  fail("Expected context to be k3d-otel-basic")
+
+include("../../cluster-infra/cert-manager/Tiltfile")
```
Comment (adrielp) [inline apps/default/Tiltfile:4]: These includes are for the ability to do depends on right?

File: cluster-infra/jaeger-operator/kustomization.yaml [hunk 3]
```diff
@@ -6,7 +6,8 @@ namespace: observability
 
 resources:
   - ../namespace
-  - https://github.com/jaegertracing/jaeger-operator/releases/download/v1.57.0/jaeger-operator.yaml
+  - localized.yaml
```
Comment (adrielp) [inline cluster-infra/jaeger-operator/kustomization.yaml:9]: For all these things, we should have an upstream file that points upstream, then a localize should be ran in workflows, and likely an edit. Basically what the monorepo does. That way we can still get dep updates.

File: apps/default/Tiltfile (hunk 2 above)
Comment (ESmithaaS) [reply apps/default/Tiltfile:4]: @adrielp, you're correct. In this particular scenario the default/basic stack requires Cert-Manager, OpenTelemetry Operator, and Jaeger Operator. When `tilt up` is run against this file, Tilt will merge the current file with the included ones and respect the "resource_deps" of all combined Tiltfiles.

File: cluster-infra/jaeger-operator/kustomization.yaml (hunk 3 above)
Comment (ESmithaaS) [reply cluster-infra/jaeger-operator/kustomization.yaml:9]: @adrielp, it's a little different with these resource references in the kustomization yaml. The monorepo is localizing kustomizations from a remote repository with the `kustomize localize` command. That only works with remote git repositories that have a kustomization yaml. 

Whereas what you have today are just download links to a standard manifest file.

**Review by adrielp — APPROVED** [review]

**Pull request merged**